client/signup/steps/import/ready/platform-details.tsx

```diff
@@ -24,7 +24,7 @@ const ImportPlatformDetails: React.FunctionComponent< DetailsProps > = ( data )
 			case 'Blogger':
 				return __( 'Importing content from Blogger' );
 			case 'Wordpress':
-				return __( 'Importing content from self-hosted WordPress to Wordpress.com' );
+				return __( 'Importing content from self-hosted WordPress to WordPress.com' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.